### DISCUSSION TITLE
jQuery 3.5.0 Security Fix

### DISCUSSION THREAD

**Discussion by uenuend** [jquery/jquery#5227] [Q&A]
Security FIX was adapted in jQuery 3.5.0, so I installed jQuery 3.6.0.
https://blog.jquery.com/2020/04/10/jquery-3-5-0-released/

However, an escape processing vulnerability was detected in the vulnerability assessment.
Isn't the escape handling security fix implemented in 3.5.0?
Also, when do you plan to schedule security fixes for escape processing?

**Comment by mgol** [comment]
There are no security issues reported for any jQuery version starting from `3.5.0` so whatever security report are you getting, it's not coming from jQuery. It must be another library or your project's own code.
